tools/gdal/.shed.yml

```diff
@@ -81,6 +81,13 @@ repositories:
       - gdal_macros.xml
       - test-data/custom.geo.json
       - test-data/ogrinfo_test1_out2.txt
+      - test-data/shp_compositefiles/shapefile.html
```

Comment:
btw. @Alanamosse have you ever tried to exclude files? 

https://github.com/kpbioteam/ewas_galaxy/blob/master/tools/minfi/.shed.yml#L14

Would that work? You could test this with the Test ToolShed.

Reply:
ping @Alanamosse ... but can wait for later.

Reply:
Yes I know the feature but not sure to understand here.
The `auto_repo` takes the needed testfiles and the `exclude` prevents the others to be in the repo as well ?
 I give it a try in test toolshed

Reply:
The idea was just, maybe its easier to exclude the few files that you don't want to have in ... Not sure. Just a thought.